prompt.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
             type: "select",
             pageSize: 2,
             choices: [
-                { message: "SASS/SCSS", name: "sass" }
+                { message: "NONE", name: "none"},
+                { message: "SASS/SCSS", name: "sass" },
+                { message: "LESS", name: "less"}
```

Comment:
Belki bunları da bu şekilde yazabiliriz? @omeraplak @aliemir @burcukaragozzz 
```suggestion
                { message: "Sass", name: "sass" },
                { message: "Less", name: "less"}
```

Reply:
benim önerim şı şekilde,
  { message: "None", name: "none"},
                { message: "Sass/Scss", name: "sass" },
                { message: "Less", name: "less"}

Reply:
Duplucate https://github.com/pankod/next-cli-core-plugins/pull/15#discussion_r557259194